src/ElasticaSearchEngine.php

```diff
@@ -281,29 +281,31 @@ public function getSearchResults($data = null, $form = null, $page = null)
             }
         }
 
+        $filtersAdded = [];
         if (isset($data['UserFilter'])) {
-            $applyDefault = false;
             $filters = $page->UserFilters->getValues();
             if (count($filters)) {
                 $queries = array_keys($filters);
                 foreach ($data['UserFilter'] as $index => $junk) {
                     if (isset($queries[$index])) {
                         $fv = explode(':', $queries[$index]);
                         $builder->addFilter($fv[0], $fv[1]);
+                        $filtersAdded[$fv[0]] = $fv[1];
                     }
                 }
             }
         }
 
-        if ($applyDefault) {
-            if ($page->InitialExpandField) {
-                // we only support a single field for now
-                $builder->setExpandFacetResults([$page->InitialExpandField => $page->ExpandedResultCount]);
-            }
+        if ($page->InitialExpandField) {
```

Comment:
Without the check as to whether we're applying defaults, does this apply the facet expansion logic _regardless_ of whether this is an unfiltered request? Another way of asking is - we don't want this expansion/bucketing happening if someone's done a search then clicked a facet filter, does that still work?

Reply:
The only change I can identify by doing the expansion in this way is that the results are now shown when a filter is selected but not facet is not (where previously results weren't being output).

In other words:
- The bucketing still occurs with nothing selected.
- The bucketing now occurs when filters are selected (the change - previously no results unless a facet is also selected).
- The bucketing still no longer occurs once a facet is selected.